editors/emacs/Makefile

```diff
@@ -1,7 +1,7 @@
 .POSIX:
-
-VERSION = 1.1.0
-NAME = lambdapi-mode
+LAMBDAPI_FILE = lambdapi-mode.el
+VERSION = $(shell cat $(LAMBDAPI_FILE) | grep ";; Version:" | sed "s/;; Version: \(.*\)/\1/")
```

Comment:
This can be simplified.
You can replace [cat file | grep string] by  [grep string file].
Idem below.
Alternative for this line: ```awk '/^;; Version: /{print $3;exit}' $(LAMBDAPI_FILE)```